catalogue_graph/src/ingestor/transformers/concept_override.py

```diff
@@ -0,0 +1,50 @@
+import csv
+import os
+from typing import TextIO
+
+from ingestor.models.concept import RawNeptuneConcept
+from ingestor.models.indexable_concept import ConceptDescription
+from ingestor.models.related_concepts import (
+    RawNeptuneRelatedConcept,
+)
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+class ConceptTextOverrideProvider:
+    overrides: dict[str, dict[str, str]] = {}
+
+    def __init__(self, overrides_csv: TextIO | None):
+        if overrides_csv:
+            self._load_overrides(overrides_csv)
+        else:
+            with open(
+                os.path.join(HERE, "label_description_overrides.csv")
+            ) as csv_file:
+                self._load_overrides(csv_file)
+
+    def _load_overrides(self, overrides: TextIO) -> None:
+        csv_reader = csv.DictReader(overrides)
+        self.overrides = {row["id"].strip(): row for row in csv_reader}
+
+    def display_label_of(
+        self, raw_concept: RawNeptuneConcept | RawNeptuneRelatedConcept
```

Comment:
This might be better as a common base class (RawBaseConcept?) that exposes the relevant @properties, but I think that can wait until we have a third class that needs to expose the same interface.